lib/propolis/src/enlightenment/hyperv/overlay.rs

```diff
@@ -175,7 +207,24 @@ pub(super) struct OverlayPage {
     pfn: Pfn,
 }
 
+impl std::fmt::Debug for OverlayPage {
```

Comment:
nit, TIOLI: maybe worth a comment stating that we don't derive this because we don't want to include the `OverlayManager`?

Reply:
Fixed in 2d23e4e.